independent-publisher-connectors/HubSpot CMS/apiProperties.json

```diff
@@ -0,0 +1,75 @@
+{
+  "connectionParameterSets": {
+    "uiDefinition": {
+      "displayName": "Authentication Type",
+      "description": "Type of authentication to be used."
+    },
+    "values": [
+      {
+        "name": "hubspot-oauth",
+        "uiDefinition": {
+          "displayName": "Use HubSpot OAuth 2.0",
+          "description": "Log in using HubSpot app client ID and secret."
+        },
+        "properties": {
+          "connectionParameters": {
+            "token": {
+              "type": "oauthSetting",
+              "oAuthSettings": {
+                "identityProvider": "oauth2",
+                "clientId": "7c3afd4c-a688-4f38-a0d9-6e4a1a1b60ed",

```

Comment:
@troystaylor Please update the client id with dummy value, like below:

`"clientId": "[[DUMMY]]"`